lib/scitree/predictions.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Scitree.Predictions do
+  import Nx.Defn
+
+  @doc """
+  Convert probabilities to predicted class.
+  This function will return the index of the class it is likely to be.
+
+  The zero value in yggdrasil is reserved for values outside the dictionary.
+
+  ## Examples
+    iex> pred = Nx.tensor([[0.01, 0.98, 0.1], [0.00, 0.01, 0.99]])
+    iex> Scitree.Predictions.probabilities_to_class(pred)
+    #Nx.Tensor<
+      s64[2]
+      [2, 3]
+    >
+  """
+  def probabilities_to_class(tensor) do
+    {size, _} = Nx.shape(tensor)
+
+    0..(size - 1)
+    |> Enum.map(&index_max_value(tensor[&1], 1))
```

Comment:
I think you want Nx.argmax here

Reply:
@jeantux argmax operates on axes as well, so maybe you can operate on the tensor matrix directly instead of Enum.map + Nx.concatenate

Reply:
Perfect!